### PR TITLE
ADCM-6157: Implement missing methods for HostProviderNode

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

File: adcm_aio_client/core/objects/cm.py [hunk 1]
```diff
@@ -41,7 +47,16 @@ def get_own_path(self: Self) -> Endpoint:
         return ("adcm",)
 
 
-class License(InteractiveObject): ...
+class License(InteractiveObject):
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/cm.py:50]: it's not interactive object (doesn't have an id)

pls sync it with @Starovoitov changes

File: adcm_aio_client/core/objects/cm.py [hunk 2]
```diff
@@ -266,6 +285,29 @@ def get_own_path(self: Self) -> Endpoint:
 class HostProvidersNode(PaginatedAccessor[HostProvider, None]):
     class_type = HostProvider
 
+    async def create(self: Self, bundle: Bundle, name: str, description: str = "") -> HostProvider:
+        if (await bundle.license).state == "unaccepted":
+            raise LicenseError
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/cm.py:290]: API will return an error if something's wrong with license

File: adcm_aio_client/core/objects/cm.py [hunk 3]
```diff
@@ -266,6 +285,29 @@ def get_own_path(self: Self) -> Endpoint:
...
+
+        prototype_data = (
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/cm.py:292]: don't we have main prototype info in bundle?

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: adcm_aio_client/core/objects/cm.py (hunk 1 above)
Comment (DanBalalan) [reply adcm_aio_client/core/objects/cm.py]: License changes reverted

File: adcm_aio_client/core/objects/cm.py (hunk 2 above)
Comment (DanBalalan) [reply adcm_aio_client/core/objects/cm.py]: removed

**Review by Sealwing — DISMISSED** [review]

**Review by a-alferov — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Pull request merged**